x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout/text_expansion_callout_logic.ts

```diff
@@ -179,7 +179,7 @@ export const TextExpansionCalloutLogic = kea<
     afterMount: async () => {
       const elserModel = await KibanaLogic.values.ml.elasticModels?.getELSER({ version: 2 });
       if (elserModel != null) {
-        actions.setElserModelId(elserModel.name);
+        actions.setElserModelId(elserModel.model_id);
```

Comment:
This part for Enterprise Search LGTM.
cc @saikatsarkar056 - you may need to update this in https://github.com/elastic/kibana/pull/171887 once this PR is merged.